src/lib/pages/tx-details/components/tx-message/TxMsgExpand.tsx

```diff
@@ -51,7 +51,7 @@ export const TxMsgExpand = ({
     case "/cosmwasm.wasm.v1.MsgStoreCode":
       msgIcon = "upload";
       content = (
-        <Flex display="inline">
+        <Flex display="inline-flex" gap={1}>
```

Comment:
Display `flex` should be fine.

```js
<Flex gap={1}>
```

Also, please update in others switch case as well. Thank!